packages/datadog-plugin-aerospike/src/index.js

```diff
@@ -0,0 +1,93 @@
+'use strict'
+
+const { storage } = require('../../datadog-core')
+const DatabasePlugin = require('../../dd-trace/src/plugins/database')
+
+class AerospikePlugin extends DatabasePlugin {
+  static get id () { return 'aerospike' }
+  static get operation () { return 'command' }
+  static get system () { return 'aerospike' }
+
+  bindStart (ctx) {
+    const { commandName, commandArgs } = ctx
+    const resourceName = commandName.slice(0, commandName.indexOf('Command'))
+    const store = storage.getStore()
+    const childOf = store ? store.span : null
+    let ns, set, key, bin, index
+    let meta = {}
+
+    if (resourceName.includes('Index')) {
+      [ns, set, bin, index] = commandArgs
+      meta = getMetaForIndex(ns, set, bin, index)
+    } else if (resourceName === 'Query') {
+      const { ns: queryNs, set: querySet } = commandArgs[2]
+      meta = getMetaForQuery({ ns: queryNs, set: querySet })
+    } else if (isKeyObject(commandArgs[0])) {
+      ({ ns, set, key } = commandArgs[0])
+      meta = getMetaForKey(ns, set, key)
+    }
+
+    const span = this.startSpan(this.operationName(), {
+      childOf,
+      service: this.serviceName({ pluginConfig: this.config }),
+      type: 'aerospike',
```

Comment:
Is this an actual type supported by our backend?

Reply:
don't know for sure, but I know both the dotnet and java tracers add it on their traces